CHANGELOG.md

```diff
@@ -10,11 +10,7 @@ This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.
 
 ### Persisted Queries w/opt-in safelisting (preview) ([PR #3347](https://github.com/apollographql/router/pull/3347))
 
-> ⚠️ **Persisted queries is an [Enterprise feature](https://www.apollographql.com/blog/platform/evaluating-apollo-router-understanding-free-and-open-vs-commercial-features/) of the Apollo Router.** It requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/).
->
-> If your organization _doesn't_ currently have an Enterprise plan, you can test out this functionality by signing up for a free [Enterprise trial](https://www.apollographql.com/docs/graphos/org/plans/#enterprise-trials).
-
-Persisted Queries gives you the tools to prevent unwanted traffic from reaching your graph.
+Persisted Queries is a forthcoming feature that helps you prevent unwanted traffic from reaching your graph.
```

Comment:
```suggestion
> ⚠️ **Persisted queries is a forthcoming [Enterprise feature](https://www.apollographql.com/blog/platform/evaluating-apollo-router-understanding-free-and-open-vs-commercial-features/) of the Apollo Router.** It will require an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/).
```